mne/io/edf/edf.py

```diff
@@ -602,7 +641,33 @@ def _read_edf_header(fname, exclude):
 
         nchan = int(_edf_str(fid.read(4)))
         channels = list(range(nchan))
-        ch_names = [fid.read(16).strip().decode('latin-1') for ch in channels]
+
+        # read in 16 byte labels and strip any extra spaces at the end
+        signal_labels = [fid.read(16).strip().decode('latin-1')
```

Comment:
```suggestion
        ch_labels = [fid.read(16).strip().decode('latin-1')
```
is more similar to mne naming